src/foam/core/types.js

```diff
@@ -489,7 +489,7 @@ foam.CLASS({
 
 foam.CLASS({
   package: 'foam.core',
-  name: 'Currency',
+  name: 'UnitValue',
```

Comment:
The plan was that `UnitValue` properties would have a method named `findUnit` that returns a Promise of a Unit. Then we would refine `UnitValue` to add a `tableCellFormatter` that calls `findUnit` to get the `Unit`, then use the `Unit`'s `format` method to get the value to display in the table cell.

Shouldn't we add the `findUnit` method here? And add the `tableCellFormatter` refinement in this PR as well?